libs/near-indexer/src/near-indexer.service.ts

```diff
@@ -479,4 +449,45 @@ export class NearIndexerService {
 
     return queryBuilder;
   }
+
+  async findAccountChangeActionsByContractName(
+    contractName: string,
+    fromBlockTimestamp?: number,
+  ): Promise<AccountChange[]> {
+    return this.buildAccountChangeActionQuery(
+      contractName,
+      fromBlockTimestamp,
+    ).getMany();
+  }
+
+  private buildAccountChangeActionQuery(
+    contractName: string,
+    fromBlockTimestamp?: number,
+  ): SelectQueryBuilder<AccountChange> {
+    return this.accountChangeRepository
+      .createQueryBuilder('account_change')
+      .leftJoinAndSelect('account_change.causedByReceipt', 'receipts')
+      .leftJoinAndSelect('receipts.originatedFromTransaction', 'transactions')
+      .leftJoinAndSelect(
+        'transactions.transactionAction',
+        'transaction_actions',
+      )
+      .leftJoinAndSelect('receipts.receiptActions', 'action_receipt_actions')
+      .leftJoin(
+        'execution_outcomes',
+        'execution_outcomes',
+        'execution_outcomes.receipt_id = receipts.receipt_id',
+      )
+      .where('account_change.affected_account_id like :id', {
```

Comment:
there is account `decentralbanksputnik-dao.near` that is not related to astro. so you should use `like '%.sputnik-dao.near'`

Reply:
we have to count calls to sputnik-dao.near as well

Reply:
added `like '%.sputnik-dao.near' or = 'sputnik-dao.near'`